plugins/modules/database/misc/redis.py

```diff
@@ -66,13 +70,15 @@
         type: str
     name:
         description:
-            - A redis config key.
+            - A redis config or db key.
         type: str
+        aliases: [ key ]
```

Comment:
```suggestion
```
I would avoid adding new aliases if there is not a very good reason why they should be added.